src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/IntegrationTestBase.scala

```diff
@@ -1,137 +0,0 @@
-package org.broadinstitute.dsde.rawls.integrationtest
```

Comment:
our workspace integration tests are superseded by other, better things and were too hard to fix